types/react-big-calendar/lib/addons/dragAndDrop.d.ts

```diff
@@ -0,0 +1,14 @@
+import BigCalendar, { BigCalendarProps, Event, stringOrDate } from '../../index';
+import React = require('react');
+
+interface withDragAndDropProps<TEvent> {
+  onEventDrop?: (args: { event: TEvent, start: stringOrDate, end: stringOrDate, allDay: boolean }) => void;
+  onEventResize?: (args: { event: TEvent, start: stringOrDate, end: stringOrDate, allDay: boolean }) => void;
+  resizable?: boolean;
+}
+
+declare class DragAndDropCalendar<TEvent extends Event = Event, TResource extends object = object>
+  extends React.Component<BigCalendarProps<TEvent, TResource> & withDragAndDropProps<TEvent>> {}
+
+declare function withDragAndDrop(calendar: typeof BigCalendar): typeof DragAndDropCalendar;
+export = withDragAndDrop;
```

Comment:
Prefer `export default`. This is not the same as https://github.com/intljusticemission/react-big-calendar/blob/8fefeee393d7173a37734da7e34af5da64959214/src/addons/dragAndDrop/index.js#L2

Reply:
Done 👍